src/tests/pull/project-state.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Priklad ako sa bude definovat stav projektu vo funkcionalnom teste -> este to nie je implementovane, bolo by toho v tomto PR uz vela ... ale pre predstavu naco sluzia `fixtures/configurations/*.json`.